src/app/personside/infotabs/meldinger/traadvisning/verktoylinje/journalforing/VelgSak.tsx

```diff
@@ -97,18 +89,21 @@ export function sakKategori(sak: JournalforingsSak): SakKategori {
 function VelgSak(props: Props) {
     const fnr = useSelector(fnrSelector);
     const valgtKategori = useFieldState(SakKategori.FAG);
-    const gsakSaker: FetchResult<Array<JournalforingsSak>> = JournalforingUtils.useSammensatteSaker(fnr);
-    const psakSaker: FetchResult<Array<JournalforingsSak>> = JournalforingUtils.usePensjonSaker(fnr);
-
-    const saker = getSaker(gsakSaker, psakSaker);
-    const fordelteSaker = fordelSaker(saker);
+    const result: FetchResult<Result> = JournalforingUtils.useSaker(fnr);
 
-    const pending = isPending(gsakSaker) || isPending(psakSaker);
-
-    if (pending) {
+    if (isPending(result)) {
         return <Spinner type="XL" />;
+    } else if (getErrors(result).length > 0) {
```

Comment:
Man må også sjekke `hasError(result)` for å sjekke om selve api-kallet feilet, det er ikke nok å sjekke `feiledesystemer`.